include/xsimd/arch/xsimd_avx512bw.hpp

```diff
@@ -410,14 +410,14 @@ namespace xsimd
             batch<T, A> xx;
             if (N & 1)
             {
-                alignas(32) uint64_t buffer[8];
+                alignas(64) uint64_t buffer[8];
```

Comment:
Occasionally segfaults from `slide` tests.
`_mm512_{load,store}_epi{32,64}` requires **64** bytes alignment, otherwise a general-protection exception may be generated.
https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm512_load_epi&ig_expand=4241

Reply:
We should be using `A::alignment()` here instead, right?

Reply:
Yes. Will update.

Reply:
Done